[2024-03-18T06:18:53.099+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: sftp_sync.sync_files_task manual__2024-03-18T06:18:40.468656+00:00 [queued]>
[2024-03-18T06:18:53.137+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: sftp_sync.sync_files_task manual__2024-03-18T06:18:40.468656+00:00 [queued]>
[2024-03-18T06:18:53.139+0000] {taskinstance.py:2193} INFO - Starting attempt 1 of 2
[2024-03-18T06:18:53.175+0000] {taskinstance.py:2217} INFO - Executing <Task(PythonOperator): sync_files_task> on 2024-03-18 06:18:40.468656+00:00
[2024-03-18T06:18:53.209+0000] {standard_task_runner.py:60} INFO - Started process 251 to run task
[2024-03-18T06:18:53.233+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'sftp_sync', 'sync_files_task', 'manual__2024-03-18T06:18:40.468656+00:00', '--job-id', '72', '--raw', '--subdir', 'DAGS_FOLDER/sftp_sync_between_source_and_target.py', '--cfg-path', '/tmp/tmpsflyww4a']
[2024-03-18T06:18:53.239+0000] {standard_task_runner.py:88} INFO - Job 72: Subtask sync_files_task
[2024-03-18T06:18:53.405+0000] {task_command.py:423} INFO - Running <TaskInstance: sftp_sync.sync_files_task manual__2024-03-18T06:18:40.468656+00:00 [running]> on host fa05d1b87173
[2024-03-18T06:18:53.578+0000] {taskinstance.py:2513} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='sftp_sync' AIRFLOW_CTX_TASK_ID='sync_files_task' AIRFLOW_CTX_EXECUTION_DATE='2024-03-18T06:18:40.468656+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-03-18T06:18:40.468656+00:00'
[2024-03-18T06:18:53.582+0000] {ssh.py:301} WARNING - No Host Key Verification. This won't protect against Man-In-The-Middle attacks
[2024-03-18T06:18:53.643+0000] {transport.py:1909} INFO - Connected (version 2.0, client OpenSSH_8.9p1)
[2024-03-18T06:18:54.777+0000] {transport.py:1909} INFO - Authentication (password) successful!
[2024-03-18T06:18:54.952+0000] {sftp.py:169} INFO - [chan 0] Opened sftp connection (server version 3)
[2024-03-18T06:18:54.970+0000] {ssh.py:301} WARNING - No Host Key Verification. This won't protect against Man-In-The-Middle attacks
[2024-03-18T06:18:55.136+0000] {transport.py:1909} INFO - Connected (version 2.0, client OpenSSH_8.9p1)
[2024-03-18T06:18:56.242+0000] {transport.py:1909} INFO - Authentication (password) successful!
[2024-03-18T06:18:56.395+0000] {sftp.py:169} INFO - [chan 0] Opened sftp connection (server version 3)
[2024-03-18T06:18:56.407+0000] {python.py:202} INFO - Done. Returned value was: Synced 0 new files.
[2024-03-18T06:18:56.577+0000] {taskinstance.py:1149} INFO - Marking task as SUCCESS. dag_id=sftp_sync, task_id=sync_files_task, execution_date=20240318T061840, start_date=20240318T061853, end_date=20240318T061856
[2024-03-18T06:18:56.701+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-03-18T06:18:56.837+0000] {taskinstance.py:3312} INFO - 0 downstream tasks scheduled from follow-on schedule check
